var/spack/repos/builtin/packages/eztrace/package.py

```diff
@@ -35,6 +36,15 @@ class Eztrace(Package):
     depends_on("cmake@3.1:", type="build", when="@2.0:")
     depends_on("otf2", when="@2.0:")
 
+    variant("starpu", default=False, description="Enable StarPU support")
```

Comment:
Do these variants have any effect if version is < 2.1 ?

Reply:
Indeed. These variants are only available with eztrace >=2.1

Reply:
I changed the variant to enable it only for >+2.1